### PR TITLE
packaging for tensile host library

### PR DESCRIPTION
**Pull request by wbgilmartin**
Added elements for packaging the tensile host "new client" libraries.

Updated CMakeFiles to enable packaging.
Updated tensile_host.cpp to find default install location.

### CODE REVIEW

File: library/src/tensile_host.cpp [hunk 1]
```diff
@@ -211,6 +213,31 @@ static auto GetTensileInputs(const RocblasContractionProblem<T, U, V>& problem)
     return inputs;
 }
 
+bool TestPath(std::string path)
+{
+  struct stat st;
+  if (stat(path.c_str(), &st) == 0)
```
Comment (leekillough) [inline library/src/tensile_host.cpp]: `access()` is probably better than `stat()`. Also, should use `const std::string& path`, like:

```
inline bool TestPath(const std::string& path)
{
    return access(path.c_str(), R_OK) == 0;
}
```

**Comment by leekillough** [comment]
There are some `clang-format` errors. To fix them, run `clang-format -i <file>` on your `.cpp` file, or run `./install.sh -h` which will install `pre-commit` in `.git/hooks` and will automatically format all file types before commit.

I already merged your branch to my working branch, since I'm working rapidly on getting the new client working with GEMM_EX functions, which require runtime type dispatch.

File: library/src/tensile_host.cpp [hunk 2]
```diff
@@ -235,12 +257,18 @@ struct TensileHostImpl : TensileHost
...
                 path.resize(strlen(path.c_str()));
-                path += "/../../Tensile/library";
             }
             else
             {
-                path = "/opt/rocm"; // TODO: Need to set default
+                path = "/opt/rocm/rocblas/lib"; 
             }
+	    // Find the location of the libraries
+            if (TestPath(path + "/../../Tensile/library"))
```
Comment (saadrahim) [inline library/src/tensile_host.cpp]: This assumes library is built in the same folder as the source. This is not necessarily true.

**Comment by leekillough** [comment]
I already merged your changes with mine, which has another PR open, so I'm closing this.

**Pull request closed without merging**